src/cmd/create.go

```diff
@@ -693,9 +689,11 @@ func pullImage(image, release string) (bool, error) {
 	}
 
 	if promptForDownload {
-		fmt.Println("Image required to create toolbox container.")
+		if _, err := podman.ImageExists(imageFull); err != nil {
+			fmt.Println("Image required to create toolbox container.")
+		}
 
-		prompt := fmt.Sprintf("Download %s (500MB)? [y/N]:", imageFull)
+		prompt := fmt.Sprintf("Download %s ? [y/N]:", imageFull)
```

Comment:
It was a [deliberate](https://github.com/containers/toolbox/issues/134) [choice](https://github.com/containers/toolbox/commit/b718fbdcda74b0f98ec5035e9a41809132f2da05) to show the size of the image.

While the number doesn't exactly match the size of the image, I don't think it's misleading. The images have always been around the 500MB mark. The Fedora 33 image is 525MB and the F35 image is 497MB. In the past they were slightly smaller, but still above 450MB.

It doesn't matter that much if the image is exactly 500MB or 400MB or whatever. The main point is that they are big, and it's important for the user to know how big before they commit to that amount of network I/O.

Everybody loves to complain about the hard-coded number, but nobody ever wants to sit down and get the right size with Skopeo. :)

Reply:
Okay that is fair. I guess it is not that important, but it does bother me each time I see it.

I think it is misleading in the sense that I thought the compressed download is maybe in the order of ~200MB??
I am not sure, am I way off here?  So how about writing `Image size` instead of `Download`?
(I just checked with skopeo manually and the current fedora-toolbox:35 `Size` is 339,350,428 bytes.)

Anyway this is just a cosmetic thing, so I am also okay to drop this commit and we could revisit it later.

Reply:
Those numbers that I were quoting were from `podman images`.

As another data point, the new `registry.access.redhat.com/ubi8/toolbox:8.5` image for RHEL 8.5 is 558MB, and the equivalent for RHEL 9 Beta is 591MB.

It's likely that the actual amount of data downloaded over the wire is less than the size reported by `podman images`, because it might be more compressed over the wire than when stored.  The numbers reported by `podman pull` seem much less than the `podman images` numbers, but I am not sure how to interpret them. I never dug into the code to convince myself exactly what's happening here.

What are the semantics of the size reported by Skopeo? Is it the amount being sent over the wire?

It will be good if someone could dig into this a bit and figure out exactly what's going on here.

As long as we are stuck with an estimate, it's better if that estimate is on the higher side. eg., if we say 300MB and then the download ends up being 450MB, and it blows the data limit of something by that extra 150MB, then the user won't be happy. :)

Hence, 400MB or 500MB doesn't matter that much, because as we begin to support more and more distributions, the image sizes will vary by that much anyway. However, if we can convince ourselves that we are only downloading in the ball park of 100MB or 150MB, then that might be something, because then it would be five times smaller. :)

Reply:
`Image size` versus `Download` seems confusing.

The only thing the user really needs to know is that we are going to download a lot (for some vague definition of *lot*) of data over the network. The approximate number is there to give some idea of what *lot* means. Everything else is only going to be confusing. eg., see all the confusion around Flatpak download sizes, progress bars, etc..

Of course, it would be good to improve our estimate, but that seems to need some digging. :)

Other than, we are also tight on screen real estate. We should try to fit most strings into a 80 character width terminal, and this string is already quite wide, and the length of the URL is outside our control:
```
Download registry.fedoraproject.org/fedora-toolbox:34 (500MB)? [y/N]:
```

It's not the most important thing, and in some cases we really don't have an option (eg., `toolbox list`), but still.

Reply:
Okay, thanks, I agree with you - it makes sense